eng/pipelines/coreclr/templates/build-job.yml

```diff
@@ -124,6 +124,10 @@ jobs:
         parameters:
           installDotnet: true
 
+    # Build CoreCLR Diagnostic tools needed by the native runtime build
+    - script: $(Build.SourcesDirectory)$(dir)build$(scriptExt) -subset clr.diagtools $(crossArg) -arch $(archType) -c $(buildConfig) $(officialBuildIdArg) -ci
```

Comment:
I think we need to pass the -bl flag with a custom name so we generate a binlog for this step.